### PR TITLE
Bug 1905329: oauth-apiserver deployment: add cpu and memory requests

### PR DESCRIPTION
**Pull request by stlaz**
The number is arbitrary, modify if it proves wrong.

### CODE REVIEW

File: bindata/oauth-apiserver/deploy.yaml [hunk 1]
```diff
@@ -39,6 +39,10 @@ spec:
           command: ['sh', '-c', 'chmod 0700 /var/log/oauth-apiserver && touch /var/log/oauth-apiserver/audit.log && chmod 0600 /var/log/oauth-apiserver/*']
           securityContext:
             privileged: true
+          resources:
+            requests:
+              cpu: 15m
+              memory: 50Mi
```
Comment (sttts) [inline bindata/oauth-apiserver/deploy.yaml:45]: what do we have in openshift-apiserver?

Have you checked a real cluster? Console should show values.

Comment (stlaz) [reply bindata/oauth-apiserver/deploy.yaml:45]: This specific container just comes and goes so it's hard to retrieve what it really attempts to request. I wasn't able to get that with `oc adm top pod --containers`, nor with the console.

The openshift-apiserver does not seem to be setting the requests here either.

Comment (sttts) [reply bindata/oauth-apiserver/deploy.yaml:45]: oh, didn't see it's not the main container. All fine.

**Pull request merged**